xla/service/gpu/tests/gpu_fused_mha_test.cc

```diff
@@ -1266,16 +1266,11 @@ class FlashAttentionBMMScaleSlidingWindowMaskSoftmaxBMM
 class FlashAttentionBMMScalePaddingMaskSoftmaxBMMF8
     : public MultiHeadedAttentionTest {
  protected:
-  void TestImpl_Flash_Attention_Inference_BMM1_NoMask_Softmax_BMM2_F8() {
-    if (skip_reason_) GTEST_SKIP() << *skip_reason_;
-    if (GetDnnVersionInfoOrDefault(backend().default_stream_executor()) <
-        se::dnn::VersionInfo(9, 1, 0)) {
-      GTEST_SKIP() << "Flash Attention requires cuDNN >= 9.1.0.";
-    }
-    XlaBuilder builder(TestName());
-    std::string hlo_string_ref =
+  const std::string GetModuleFlashAttentionBMMScaleSoftmaxBMMCommonRef() {
```

Comment:
I believe there is no reason for this function to be a method of the test class. Please put this in a separate function that is defined right before the `TEST_...` macros that use it. See other comments as well.